### PR TITLE
feat: add powtargetspacing to dash.conf vars

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

Dash Core now supports the `powtargetspacing` setting

## What was done?

- Add powtargetspacing

## How Has This Been Tested?

On `devnet-jack-daniels`

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Comment by ktechmidas** [comment]
LGTM

**Review by ktechmidas — APPROVED** [review]

**Comment by ktechmidas** [comment]
(although, do we define this manually in the config file? Nothing in bin/generate for this yet?)

File: ansible/roles/dashd/templates/dash.conf.j2 [hunk 1]
```diff
@@ -90,6 +90,9 @@ llmqinstantsenddip0024=llmq_devnet
 minimumdifficultyblocks={{ dashd_minimumdifficultyblocks }}
 highsubsidyblocks=500
 highsubsidyfactor=10
+{% if dashd_powtargetspacing is defined %}
```
Comment (nmarley) [inline ansible/roles/dashd/templates/dash.conf.j2:93]: @strophy Is this only for devnets?

Comment (strophy) [reply ansible/roles/dashd/templates/dash.conf.j2:93]: @PastaPastaPasta @ogabrielides ^^

**Comment by strophy** [comment]
> (although, do we define this manually in the config file? Nothing in bin/generate for this yet?)

I didn't add this to the config generator because there is no good default value. For most "normal" devnets this should not be defined, so I think the correct approach is to conditionally include it in config only if defined manually.

File: ansible/roles/dashd/templates/dash.conf.j2 (hunk 1 above)
Comment (ogabrielides) [reply ansible/roles/dashd/templates/dash.conf.j2:93]: Correct

**Review by nmarley — APPROVED** [review]
utACK

**Pull request merged**